web/packages/teleport/src/Notifications/Notification.tsx

```diff
@@ -83,77 +87,64 @@ export function Notification({
 
   return (
     <>
-      <Wrapper>
-        <Container
-          className={notification.clicked ? 'clicked' : 'new'}
-          onClick={() => onNotificationClick()}
-        >
-          <GraphicContainer>
-            <MainIconContainerWrapper>
-              <MainIconContainer className={content.type}>
-                {content.icon}
-              </MainIconContainer>
-            </MainIconContainerWrapper>
-            <AccentIconContainerWrapper>
-              <AccentIconContainer className={content.type}>
-                <AccentIcon />
-              </AccentIconContainer>
-            </AccentIconContainerWrapper>
-          </GraphicContainer>
-          <ContentContainer>
-            <ContentBody>
-              <Text>{content.title}</Text>
-              {content.onQuickActionClick && (
-                <ButtonSecondary
-                  css={`
-                    height: 32px;
-                    text-transform: none;
-                  `}
-                >
-                  {content.quickActionButtonText}
-                </ButtonSecondary>
-              )}
-            </ContentBody>
-            <SideContent>
-              <Text
+      <Container clicked={notification.clicked} onClick={onNotificationClick}>
+        <GraphicContainer>
+          <MainIconContainer type={content.type}>
+            <content.icon size={18} />
+          </MainIconContainer>
+          <AccentIconContainer type={content.type}>
+            <AccentIcon size={10} />
+          </AccentIconContainer>
```

Comment:
Do we still need containers? Perhaps we could style the icons directly.

Reply:
I think it's better to keep the containers, since these styles are there for the "frame" containing the icon (the square with the background and border) and I find this much cleaner than doing it directly on the svg icon element.